src/axom/quest/SamplingShaper.hpp

```diff
@@ -456,6 +456,53 @@ class SamplingShaper : public Shaper
   //@}
 
 public:
+  /**
+   * \brief Import an initial set of material volume fractions before shaping
+   *
+   * \param [in] initialGridFuncions The input data as a map from material names to grid functions
+   * 
+   * The imported grid functions are interpolated at quadrature points and registered
+   * with the supplied names as material-based quadrature fields
+   */
+  void projectInitialVolumeFractions(
+    const std::map<std::string, mfem::GridFunction*> initialGridFunctions)
+  {
```

Comment:
Why is this called `projectInitialVolumeFractions` instead of `importInitialVolumeFractions`?

Reply:
`project` because the data comes in as grid functions (i.e. the high order equivalent of cell-centered data) and is then projected/resampled/interpolated at a different set of quadrature point locations within the cell.

I can change it to `import` if that seems clearer.
More directly: I will change it to `import` unless others object.

Reply:
done